main.md

```diff
@@ -861,10 +856,9 @@ Figure: HTTP 401 Response to a Protected Resource Request with an Invalid Token
 
 Protected resources simultaneously supporting both the `DPoP` and `Bearer` 
 schemes need to update how evaluation of bearer tokens is performed to prevent 
-downgraded usage of a DPoP-bound access tokens. 
-Specifically, such a protected resource MUST reject an access
-token received as a bearer token per [@!RFC6750], if that token is
-determined to be DPoP-bound. 
+downgraded usage of a DPoP-bound access tokens.
```

Comment:
access tokens -> access token.

Reply:
and done f51a9d998120a95c0f6e8fa64195f566293ab7e2